### ISSUE TITLE
we need to add upload and download in zos help container 

### ISSUE THREAD

**Issue by Dina-Abd-Elrahman** [threefoldtecharchive/zosctl#49]
- need to add upload and download for the output of ```zos help container```

# ZOS Version:
619b56a321c99b7c7ce333b2143dc0250b9514d1

# Verification Steps:
-  ./zos help container
```
  zos container new --name=<container> --root=<rootflist> [--hostname=<hostname>] [--sshkey=<sshkey>] [--privileged] [--ssh]
    creates a new container 

  zos container inspect
    inspect the current running container (showing full info)

  zos container info
    shows summarized info on running containers
  zos container list
    alias to `zos container info`

  zos container <id> inspect
    shows detailed information on container 

  zos container <id> info
    show summarized container info

  zos container <id> delete
    deletes containers

  zos container <id> zerotierinfo
    shows zerotier info of a container

  zos container <id> zerotierlist
    shows zerotier networks info

  zos container <id> exec <command>
    executes a command on a specific container

  zos container <id> sshenable
    enables ssh on a container

  zos container <id> sshinfo
    shows sshinfo to access container

  zos container <id> shell
    ssh into a container
```

**Comment by xmonader** [comment]
fixed in 38cd869ebe2bf9d2ee07ab36c142509201db8858

**Comment by siddiquagig** [comment]
Verified

- ./zos help container
```
zos container <id> upload <file> <dest>
    upload <file> to <dest> on container <id>

  zos container <id> download <file> <dest>
    download <file> to <dest> from container <id>

  zos container upload <file> <dest>
    uploads the <file> to <dest> on the last created container by zos

  zos container download <file> <dest>
    downloads <file> to <dest> from the last created container by zos 
```
